### ISSUE TITLE
python3 compatibility

### ISSUE THREAD

**Issue by Gaurav-t** [Merino/django-admin-tools#1]
I'm migrating my project to python3, Is django-admin-tools - 0.5.2 works with python3?

**Comment by lzanuz** [comment]
Yes, It works. I'm using it with python3. There is a new version now. 0.6.0.
